packages/rdma-core/rdma-core.spec

```diff
@@ -21,6 +21,8 @@ Requires: %{_cross_os}libnl
 
 %package devel
 Summary: RDMA core development libraries and headers
+Requires: %{name}
```

Comment:
Thanks for the fix! Out of curiosity, how did you find the bug?

Reply:
> Thanks for the fix! Out of curiosity, how did you find the bug?

I copied `rdma-core.spec` as the starting point for `libjson-c.spec` since they both use `cmake`, and then got a linker error because the library was missing even though `libjson-c-devel` was installed. 😀